spec.html

```diff
@@ -22719,7 +22737,7 @@ <h1>ParseModule ( _sourceText_, _realm_, _hostDefined_ )</h1>
                     1. Append _ee_ to _localExportEntries_.
                   1. Else this is a re-export of a single name,
                     1. Append the ExportEntry Record {[[ModuleRequest]]: _ie_.[[ModuleRequest]], [[ImportName]]: _ie_.[[ImportName]], [[LocalName]]: *null*, [[ExportName]]: _ee_.[[ExportName]] } to _indirectExportEntries_.
-              1. Else if _ee_.[[ImportName]] is `"*"`, then
+              1. Else if _ee_.[[ImportName]] is `"*"` and _ee_.[[ExportName]] is *null*, then
```

Comment:
should this say `is not *true*` instead?

Reply:
I'm don't think so -- I'm using a check on `[[ExportName]]` to differential between the last two record types in this table: https://spectranaut.github.io/proposal-export-ns-from/#table-42

So, `[[ExportName`]] is never `true`, it's either null or a string. Which makes me realize this table should be updated: https://spectranaut.github.io/proposal-export-ns-from/#table-41

To explain a bit more: If `[[ExportName]]` is null, this record should go into `[[StarExportEntries]]`, if it is set, it should go into the `[[IndirectExportEntries]]` (an related question -- do you think the editorial text in this table should be updated? https://spectranaut.github.io/proposal-export-ns-from/#table-38 )

Reply:
Ah, i confused it with the new slot you added that’s a boolean. As-is this is fine,